testsuite/performance/README.provisioning-parameters.md

```diff
@@ -1,48 +1,121 @@
+
+### Monitoring
+
+| Category    | Setting                       | Property                   | Default Value   |
+|-------------|-------------------------------|----------------------------|-----------------|
+| Docker      | Allocated CPUs                | `monitoring.docker.cpuset` | `0`             |
+
+
+## Note on Docker settings
 
 By default, there are 4 CPU cores allocated: core 0 for monitoring, core 1 for database (MariaDB), and cores 2 and 3 for Keycloak server.
 Default memory limits for database and Keycloak server are 2g. The `cpuset` and `memlimit` parameters set here are set to `cpuset` and
 `mem_limit` parameters of docker-compose configuration. See docker-compose documentation for meaning of the values. How to set the parameters
 correctly depends on number of factors - number of cpu cores, NUMA, available memory etc., hence it is out of scope of this document.
 
-| Container   | Setting                       | Property                        | Default value                                         |
-|-------------|-------------------------------|---------------------------------|-------------------------------------------------------|
-| Keycloak    | Allocated CPUs                | `keycloak.docker.cpuset`        | 2-3                                                   |
-|             | Allocated CPUs for DC1        | `keycloak.dc1.docker.cpuset`    | 2-3                                                   |
-|             | Allocated CPUs for DC2        | `keycloak.dc2.docker.cpuset`    | 2-3                                                   |
-|             | Available memory              | `keycloak.docker.memlimit`      | 2g                                                    |
-| MariaDB     | Allocated CPUs                | `db.docker.cpuset`              | 1                                                     |
-|             | Available memory              | `db.docker.memlimit`            | 2g                                                    |
-| Monitoring  | Allocated CPUs                | `monitoring.docker.cpuset`      | 0                                                     |
+### Example CPU Settings
+
+| HW          | Development Machine  | "Fat Box"    |
+|-------------|----------------------|--------------|
+| Cores       | 4                    | 48           |
+| NUMA Nodes  | 0-3                  | 0-23, 24-47  |
+
+#### Cluster
+
+| Setting                            | Development Machine  | "Fat Box"                   |
+|------------------------------------|----------------------|-----------------------------|
+| `keycloak.docker.cpusets`          | 2-3                  | 4-7 8-11 12-15 ... 44-47    |
+| `db.docker.cpuset`                 | 1                    | 1                           |
+| `lb.docker.cpuset`                 | 1                    | 2                           |
+| `infinispan.docker.cpuset`         | 1                    | 3                           |
```

Comment:
`infinispan.docker.cpuset` needs to be removed. Infinispan server is only present in the Cross-DC deployment.